### PR TITLE
Fixes bawiki's name and adds bswiki.

### PR DESCRIPTION
**Pull request by halfak**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -94,6 +96,10 @@ tuning_reports: \
 		wikidatawiki_tuning_reports \
 		zhwiki_tuning_reports
 
+touch:
```
Comment (Ladsgroup) [inline Makefile:99]: Add this into the template otherwise it'll be gone by the next run

**Comment by Ladsgroup** [comment]
One thing and it's good to go.

**Comment by halfak** [comment]
WHat's the one thing?  I don't see any notes.  @Ladsgroup :P

**Comment by Ladsgroup** [comment]
Oops, I forgot to submit them :D

File: Makefile (hunk 1 above)
Comment (halfak) [reply Makefile:99]: I didn't add this.  it must have been part of the template already.

Comment (halfak) [reply Makefile:99]: See https://github.com/wiki-ai/editquality/commit/979dc7fe75c3f11ef1f94697baae72bb907e145f

**Pull request merged**